### ISSUE TITLE
Add more supported models

### ISSUE THREAD

**Issue by triumfas** [mitch-dc/volkswagen_we_connect_id#201]
**Is your feature request related to a problem? Please describe.**
Integration works with other VW models too. Not all sensors do work, but most of the information is shown.

**Describe the solution you'd like**
Add "Polo" and "Tiguan" as supported vehicles

**Additional context**
SUPPORTED_VEHICLES = ["ID.3", "ID.4", "ID.5", "ID. Buzz", "Polo", "Tiguan"]

Polo (Taigo) and Tiguan work for me (need some modification inside, but still)

**Comment by Chesstimation** [comment]
"e-Golf" and "e-up!" also work!

**Comment by pietsab** [comment]
"Passat GTE" also work

**Comment by NeWoKiKi** [comment]
> "Passat GTE" also work

I added "Passat GTE" but it doesn't work for me, did I miss something ?

`SUPPORTED_VEHICLES = ["ID.3", "ID.4", "ID.5", "ID. Buzz", "Passat GTE"]`

**Comment by pietsab** [comment]
> > "Passat GTE" also work
> 
> I added "Passat GTE" but it doesn't work for me, did I miss something ?
> 
> `SUPPORTED_VEHICLES = ["ID.3", "ID.4", "ID.5", "ID. Buzz", "Passat GTE"]`

after the change in the init file, I add the integration. I have to type my account mail and password in the boxes and after, I guess some minutes, I get my Passat as an device

**Comment by NeWoKiKi** [comment]
I've just tried to remove integration, check that "Passat GTE" is added in init file and re add it... even after waiting some minutes I don't have any device
I don't understand :o

**Comment by pietsab** [comment]
first, i need to delete all other integrations and repos related to Volkswagen. Then in HACS  I install the VW Connect ID repo. After that I add in the init__py file "Passat GTE". Then check the configuration and reboot HA. Now I can add the integration VW Connect ID and is still running. There are some errors in the LOG

**Comment by NeWoKiKi** [comment]
I followed exactly what you did ; delete all integrations, rebooted HA, installed VW Connect ID, edited init file, rebooted and added VW integration but it doesn't dicover my Passat GTE :(

In logs I have this
`Setup of platform volkswagen_we_connect_id is taking longer than 60 seconds. Startup will proceed without waiting any longer.`

**Comment by NeWoKiKi** [comment]
Do we agree that correct application is this one ?

https://play.google.com/store/apps/details?id=com.volkswagen.weconnect

I can see my Passat GTE in this application

**Comment by tomm1ed** [comment]
Could you also add support for the e-Golf? the model returned is:

e-Golf

**Comment by pietsab** [comment]
> Could you also add support for the e-Golf? the model returned is:
> 
> e-Golf

i do it by my own and follow the instruction https://github.com/mitch-dc/volkswagen_we_connect_id/compare/main...danielszilagyi:volkswagen_we_connect_id:main#diff-31afb15cb0fb003e221a5d2bc6f981b22949d1fdfa6dc037b064b767802c7a0d

in addition to your e-Golf perhapse you can use this information https://github.com/robinostlund/homeassistant-volkswagencarnet/issues/493#issuecomment-1699646742

**Comment by pietsab** [comment]
> Do we agree that correct application is this one ?
> 
> https://play.google.com/store/apps/details?id=com.volkswagen.weconnect
> 
> I can see my Passat GTE in this application

Yes, I agree. I'm not a programmer or anything. So I hope you understand that I can only provide limited help to you. I tried several names of Passat myself - "Passat" / "Passat Variant" and my solution is "Passat GTE"

**Comment by andre68723** [comment]
the underlying library used confirms the GTE - see https://github.com/tillsteinbach/WeConnect-python/blob/607c579c888a1d1ee7a129049b4f53adc75f46db/README.md?plain=1#L34

**Comment by skallberg** [comment]
I have added "Passat GTE" to the config but it doesn't work for me.
I have a Passat GTE 2018
Get the following error:

ERROR (MainThread) [homeassistant.config_entries] Error setting up entry Volkswagen We Connect ID for volkswagen_we_connect_id
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 388, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/volkswagen_we_connect_id/__init__.py", line 81, in async_setup_entry
    await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 1367, in async_forward_entry_setups
    await asyncio.gather(
asyncio.exceptions.CancelledError

**Comment by ryanjdillon** [comment]
> "e-Golf" and "e-up!" also work!

Are you able to control climatization once you've added them? I was also able to get my e-Golf sensor up, but the services defined in `services.yaml` and `__init__.py` do not appear. 

I've tested climatization control via the `weconnect` SDK, and it works, so this should be possible... can't really see why these services wouldn't succeed from glancing at the code.

**Comment by skallberg** [comment]
Are there any plans to include full Passat GTE support?
I have a 2018 Passat GTE and I can not get this integration to work.

**Comment by clabbo** [comment]
Hi. I just wanted to mention that "Golf GTE" also works if it is added to the SUPPORTED_VEHICLES (although it does not create any services).
On a side note, I did have a lot of issues with timeouts and "too many requests" for my VINs. It seems to have disappeared after changing the update_interval=timedelta(seconds=45) to 300. The updates are polled every 5 minutes which is good enough for me. I suppose other integrations I have also add to the count, like Tibber and the Volkswagen app itself.

**Comment by vicpuntnl** [comment]
I'm trying to get my Passat GTE in HA. I'm as far as that I added the integration and logged in to my VW-account. Bu then The integration does have 'No devices or entities'. My Passat is a 2020-model.
I was wondering @pietsab: what model(year) do you got working? And do you have/need a payed ConnectPlus subscription to see anything anyway?

**Comment by pietsab** [comment]
Hello, I have entered “Passat GTE” in the supported vehicles as described above and yes, I use the payment service.
There is a [second integration](https://github.com/robinostlund/homeassistant-volkswagencarnet) for HA that also works without changing the code.
No matter which integration I try to use, when I am loading the vehicle, the report from VW's backend is stopped after a short time. Only when I stop the respective integration again in HA is my vehicle available again after a short time in the front end of the VW ID app.
The bottom line is that I no longer use integration at all but only VW.
By the way - I use Tibber to set my departure time based on the electricity price.
Funny: Tibber probably uses the same backend to VW without any problems compared to the two integrations.

greetings

**Comment by uougaard** [comment]
you wrote that the reporting from VW stops after a short time, and you need to stop the integration, wait, and then start the integration again.  This could be a symptom that your setup has a too short update interval. I have a setup with approximately 2 minutes update interval, and it never stops retrieving updates, although it sometimes fails to update for short time

**Comment by vicpuntnl** [comment]
Thanks for you answer! I started paying for vw connect+, and used the "second integration". That one worked well. This integration still doesn't, but maybe I should really clean everything up first, and then install it all the way again.

**Comment by pietsab** [comment]
@uougaard: I apologize, I may have described it incorrectly.

I install and start the integration in HA. I get the values for my Passat with the initialization. If I then plug my car in to charge at some point, e.g. 0% SOC, I receive a programmed message every 5 minutes via HA that my car is charging up to approx. 65% SOC, i.e. approx. 1 hour and 20 minutes. From this point onwards, I no longer receive a message from HA. I then look in the VW app and my vehicle is no longer displayed and the tile for charging and air conditioning is empty.
I then have to stop the integration in HA so that my Passat appears again in the VW app after about 1/2 hour.

The above process or error pattern is the same for both integrations that I had already mentioned in the previous post. 

@vicpuntnl: That sounds like a plan ;-)

**Comment by lordvalium** [comment]
> "Passat GTE" also work

Really? Can you read out the SOC? I cant on a 2021 Passat GTE
